changelog-2022.08.23-5.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/qt/res/app.organicmaps.desktop.metainfo.xml b/qt/res/app.organicmaps.desktop.metainfo.xml
+index e3541249a5..4b3cd65ed0 100644
+--- a/qt/res/app.organicmaps.desktop.metainfo.xml
++++ b/qt/res/app.organicmaps.desktop.metainfo.xml
+@@ -107,6 +107,16 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2022.08.23-5" date="2022-08-23">
++     <description>
++       <ul>
++         <li>New OSM data as of August 16</li>
++         <li>Longer routes are calculated faster</li>
++         <li>Search should work way better and faster now</li>
++         <li>Reduced size of mwm files</li>
```

Comment:
• Fixed issue when some found objects were not highlighted on the map
• Removed common street search synonyms like parkway, path, line, walk, trail, etc.
• Fixed coordinates search if the query contains a newline
• Ignore non-relevant search results for coordinates
• Better search for capitals
• Treat 'emergency' as a last-choice feature type
• Added Catalan street search synonyms
• Properly filter equal streets in search results

• Improved visibility of military areas and boundaries
• Show disabled parking spaces
• Do not draw icon for residential gardens
• Render farm yard
• Add parking_entrance
• Add playground and internet cafe icon
• Add second_hand and charity
• Add table_tennis, volleyball, beachvolleyball, skateboard, chess, padel, handball, futsal, hockey, badminton, pelota
• Removed background circles from icons

Reply:
Thanks! Changing it.